adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -33,6 +33,12 @@
 import neopixel
 import touchio
 
+try:
+    from typing import Literal, Optional, Iterator
```

Comment:
clue about setup.py would be nice.  All I see is setup.py.disabled. Is that because of Pycharm?

Reply:
You'll need to update this branch to the latest `main` branch, that file has been deleted and since replaced with `setup.py` in a recent pull request.

Reply:
yeah, got that update now.  is it added as just a simple import in setup.py? or some other way?

Reply:
You add it to the list of `install_requires` in `setup()`

```python
    install_requires=[
        "adafruit-circuitpython-lis3dh",
        "adafruit-circuitpython-thermistor",
        "adafruit-circuitpython-neopixel",
        "typing-extensions~=4.0",
    ],
```

Reply:
it threw a number error after adding this line, and went back to just dying when I removed "~=4.0".   All pylint is showing me locally is that it does not like the uname function of os, in addition to the line ending errors.

Not sure if you can see what specifically is choking it here.....